SaltwaterTaffy/Ndiff.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace SaltwaterTaffy
+{
+    public static class Ndiff
+    {
+        /// <summary>
+        ///     Run the Ndiff tool to compare output files from Nmap.
+        /// </summary>
+        /// <param name="file1">Original File</param>
+        /// <param name="file2">Modified File</param>
+        /// <param name="verbose">Also show hosts and ports that haven't changed.</param>
+        /// <param name="xml">Display output in XML format.</param>
+        /// <returns>Output from Ndiff application.</returns>
+        public static string RunDiff(bool verbose, bool xml, string file1, string file2)
```

Comment:
Would prefer to use the `NmapOptions` pattern here (maybe an `NdiffOptions` object), but I'm fine leaving it like this, if this is the only functionality you need.

Reply:
That's make sense to use the NmapOptions pattern. I'll try to have another commit before the weekend with this and the other changes per your comments.